addons/docs/src/frameworks/web-components/custom-elements.ts

```diff
@@ -50,30 +50,58 @@ function mapData(data: TagItem[], category: string) {
   return (
     data &&
     data
-      .filter((item) => !!item)
+      .filter((item) => item && item.name)
```

Comment:
I added this because there where items in the loop with an empty `name`. If I look in `custom-elements.json`, I indeed see some items without a `name`: ```            {
              "type": {
                "text": "CustomEvent"
              }
            },
```